Cancelled teardowns are undesirable. Err on the side of waiting a long time.

--- netifd/proto-shell.c.orig	2019-04-23 13:28:15.157876351 +1000
+++ netifd/proto-shell.c	2019-04-23 13:29:42.902633651 +1000
@@ -197,7 +197,7 @@
 		switch (state->sm) {
 		case S_SETUP:
 			if (state->script_task.uloop.pending) {
-				uloop_timeout_set(&state->teardown_timeout, 1000);
+				uloop_timeout_set(&state->teardown_timeout, 30000);
 				kill(state->script_task.uloop.pid, SIGTERM);
 				if (state->proto_task.uloop.pending)
 					kill(state->proto_task.uloop.pid, SIGTERM);
@@ -214,7 +214,7 @@
 				snprintf(error_buf, sizeof(error_buf), "ERROR=%d", state->last_error);
 				envp[j++] = error_buf;
 			}
-			uloop_timeout_set(&state->teardown_timeout, 5000);
+			uloop_timeout_set(&state->teardown_timeout, 30000);
 			break;
 
 		case S_TEARDOWN:
